egov-pg-service/src/main/java/org/egov/pg/service/gateways/axis/AxisGateway.java

```diff
@@ -85,6 +85,9 @@ public AxisGateway(RestTemplate restTemplate, Environment environment, ObjectMap
     @Override
     public URI generateRedirectURI(Transaction transaction) {
 
+        Map metaData = transaction.getMetaData();
+        String BANK_ACCOUNT_NUMBER = (String)metaData.get("accountNumber");
```

Comment:
The constant you are fetching should be replaced from `accountNumber` to `BANK_ACCOUNT_NUMBER` and not the variable name. Also you need to have a class to get available meta data variable. Else if someone forgot to define the variable in config then there will be a `NullPointerException`. Which in actual should raise a gateway config error that the required parameter is not configured. So the method for fetching a parameter should have a additional parameter asking if this is a require parameter or not